packs/equipment-effects/effect-beastmasters-sigil-melee-weapon-greater.json

```diff
@@ -1,6 +1,6 @@
 {
     "_id": "gU3uZE2ihLnpQN0b",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/creatures/mammals/wolf-shadow-black.webp",
```

Comment:
Did you intend for the greater, major, and normal to have different icons?

Reply:
Different animals per the description, so I was going for that.